project/yuanhenglee/README.rst

```diff
@@ -0,0 +1,179 @@
+# BasiCPP Pitch: A C++ implementation for AMT(Automatic Music Transcription)
```

Comment:
You name the file as rst but use markdown in the content.  It's better to make them consistent.

Reply:
Sure! I suppose the picture issue below will be fixed after I correct the file into markdown.